examples/proto_dice/tests/proto_dice_tests.cpp

```diff
@@ -23,11 +24,58 @@ class proto_dice_tester: public game_tester {
 
         deploy_game<proto_dice_game>(game_name, game_params);
     }
+
+    double get_rtp(uint64_t run_count, const uint32_t bet_num) {
+        const auto player_name = N(player);
+        create_player(player_name);
+        link_game(player_name, game_name);
+
+        const auto to_double = [](const asset& value) -> double {
+            return double(value.get_amount()) / value.precision();
+        };
+
+        // Init balance for player and casino
+        transfer(N(eosio), player_name, STRSYM("1000000.0000"));
+        transfer(N(eosio), casino_name, STRSYM("1000000.0000"));
+
+        const double start_player_balance = to_double(get_balance(player_name));
+
+        auto graph = strategy::Graph([&](game_tester & tester, const uint32_t session_id) {
+                return strategy::Result::Continue;
```

Comment:
Can we just
```
tester.game_action(game_name, ses_id, 0, { bet_num });
return strategy::Result::End;
```
here, instead of push_child below?

Reply:
This code for example more than for real test.